src/components/LocationSelector/LocationSelectorDialog/LocationSelectorDialog.tsx

```diff
@@ -1,19 +1,62 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { bem } from '../../../utils';
-import { LocationCard } from '../../LocationCard';
+import { LocationCard, LocationCardProps } from '../../LocationCard';
 import { Text } from '../../Text';
 import { Slider } from '../../Sliders';
 import { Button } from '../../Buttons';
-import { LocationAutocomplete } from '../../LocationAutocomplete';
-import { Map } from '../../Map';
+import { LocationAutocomplete, LocationAutocompleteProps } from '../../LocationAutocomplete';
+import { Map, CircularMarker } from '../../Map';
 import { SIZES, ENTER_KEY } from '../../../constants';
 import styles from './LocationSelectorDialog.scss';
+import { LocationSelectorLocation } from '../utils';
+
+// Props for child components that we just push through with prop-drilling
+type CardPropsForSelectorDialog = Omit<
```

Comment:
Should these perhaps better be moved to `src/types/`?

Reply:
I think it is pretty specific here and won't make sense in the global space.